test-user.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+from birdyboard import *
+
+
+class TestUser(unittest.TestCase):
+
+    def test_user_creation(self):
+        self.birdy = Birdyboard()
+        self.birdy.user = self.birdy.User("bob vila", "bob_the_builder")
```

Comment:
While you do need to call `self.birdy.User()` to instantiate the user object since User is a subclass under Birdyboard, you can just declare `self.birdy.user` as `self.user`, there's no need to attach it as an attribute of `birdy`.  There's nothing _wrong_ with doing it that way, so far as I know, just wanted to mention that you don't _have_ to do it that way.